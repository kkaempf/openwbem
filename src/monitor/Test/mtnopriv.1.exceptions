InsufficientPrivilegesException   CheckException: readDirectory: insufficient privileges
PrivilegeManagerException  PrivilegeManager.* no connection to the monitor
