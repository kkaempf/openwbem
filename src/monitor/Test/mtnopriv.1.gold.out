User: owprovdr

readDirectory
Exception:
  type: PrivilegeManagerException
  msg:  CheckException: readDirectory: insufficient privileges

readDirectory
Exception:
  type: IPCIOException
